_includes/graphql/queries.md

```diff
@@ -0,0 +1,408 @@
+# Queries
+
+## Get
+
+For each class of your application's schema, Parse Server automatically generates a custom query for getting this class' objects through the GraphQL API.
```

Comment:
Not sure it needs to specifically say 'GraphQL API' all the way through the guide...
```suggestion
For each class of your application's schema, Parse Server automatically generates a custom query for getting this class' objects through the API.
```
If you agree perhaps it should be changed throughout the guide also?